UserStory1/ex4test.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+from ex4 import isPassword_valid
+
+class Testex4(unittest.TestCase):
+    
+    def test_valid_password(self):
+        self.assertTrue(isPassword_valid('Shreya123#'))
```

Comment:
Test outcome depends upon user inputs.
All input required to the test should be provided as parameters.
Update the `isPassword_valid()` function to take a list of passwords